recipes/torchbiggraph/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "torchbiggraph" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/PyTorch-BigGraph/archive/v{{ version }}.tar.gz
+  sha256: 3a79be53266beffbff6bdf4d54d7779f1b7cc5b988afd5e00880deb8e31a682b
+
+build:
+  skip: True  # [py<36]
```

Comment:
There should be entry_points defined: https://github.com/facebookresearch/PyTorch-BigGraph/blob/master/setup.cfg#L54-L63

This will probably allow it be be made noarch ad the Windows error quoted before normally just means entry_points are missing.